src/components/scripts/utilities.js

```diff
@@ -43,6 +43,7 @@ const availableSpecies = () => {
     "Mouse":{taxo: "NCBITaxon:10090", iconClass:"mapicon-icon_mouse", displayWarning: true},
     "Pig":{taxo: "NCBITaxon:9823", iconClass:"mapicon-icon_pig", displayWarning: true},
     "Cat":{taxo: "NCBITaxon:9685", iconClass:"mapicon-icon_cat", displayWarning: true},
+    "Functional Connectivity": {taxo: "FunctionalConnectivity", displayWarning:true},
```

Comment:
Remove this please, function connectivity should not be available in the chooser.